eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -47,6 +50,33 @@ $headers = @{
 
 $query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
 
+function AddLabels([int] $prNumber)
+{
+  # Adding labels to the pr.
+  if (-not $PRLabel) {
+    Write-Verbose "There are no labels added to the PR."
+    exit 0
+  }
+
+  # Parse the labels from string to array
+  $prLabels = @($PRLabel.Split(",") | % { $_.Trim() } | ? { return $_ })
+  $prLabelUri = "https://api.github.com/repos/$RepoOwner/$RepoName/issues/$prNumber"
+  $labelRequestData = @{
+    maintainer_can_modify = $true
```

Comment:
Why is this option interesting for applying labels?

Reply:
It allows pr to be modified when it i. I know we have this when PR created. Put it here to make sure the we do not change the permission after adding the label.

Reply:
It feels like it is out of place here so I would remove it unless it is needed for some reason.